proto/value.proto

```diff
@@ -5,10 +5,10 @@ package gink;
 
 message Value {
     oneof value {
-        sint64 bigint = 1;
+        sint32 integer = 1;
```

Comment:
This was the issue causing TypeScript to incorrectly interpret normal integers as big ints. I can't really tell you why this works, aside from maybe TypeScript was going down the proto list to see which data type matched the value, and int64 technically works for a smaller integer.